.coveragerc

```diff
@@ -1,6 +1,8 @@
 # .coveragerc to control coverage.py
 [run]
 source = blackbird_python/blackbird
+omit =
+    blackbird_python/blackbird/blackbird*.py
```

Comment:
These are the autogenerated parser files, which are very hard to test directly so are omitted from coverage.

Anything with filename matching `blackbird*.py` can also be ignored from this PR.